helm/charts/hpe-csi-driver/templates/hpe-csi-node.yaml

```diff
@@ -73,6 +73,16 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: CHAP_USER
```

Comment:
We still need to add values.yaml changes and CRD changes to provide these during install. you can add new type like below in values.yaml

```
iscsi:
  chap_user:
  chap_password:
```

Reply:
thanks. operators/crds and CR file has to be changed as well and new CSV should be generated..I can walk you through those steps.

Reply:
sure.